clang/test/CodeGen/SystemZ/builtin-setjmp-logjmp-backchain.c

```diff
@@ -0,0 +1,25 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 5
```

Comment:
I don't think we need a -backchain version of the test case here - what's tested here is completely identical as far as the front-end goes.  Just the other test should be fine.